src/main/views/includes/scripts.njk

```diff
@@ -6,7 +6,7 @@
 <script src="{{ asset_paths['js'] }}/application.js"></script>
 <script src="{{ asset_paths['js'] }}/detectJavascript.js"></script>
 <script src="{{ asset_paths['js'] }}/analytics.js"></script>
-<script src="{{ asset_paths['js'] }}/analytics-custom-event-sender.js"></script>
+<script src="{{ asset_paths['js'] }}/analytics-custom-event-sender.js?2017-10-11"></script>
```

Comment:
Should not be needed but preferred to add it in case. In the future asset fingerprinting should take care of it.